### ISSUE TITLE
Cargo recompiles unicornafl every time

### ISSUE THREAD

**Issue by FelixBrakel** [AFLplusplus/unicornafl#2]
When adding unicornafl to the cargo dependencies in the following way:

```toml
[dependencies]
unicornafl = { git = "https://github.com/AFLplusplus/unicornafl", version="1.1.0" }
```
It causes unicornafl to be recompiled every time. Possibly a mistake in the build.rs or maybe an error in cargo?

**Comment by wtdcode** [comment]
Sorry for my late reply. The rust bindings of unicornafl needs some polish indeed.

**Comment by FelixBrakel** [comment]
I ended up creating my own fork and commenting out the line 
```rust
println!("cargo:rerun-if-changed=unicornafl");
```
in `build.rs`, which seemed to fix the issue.

Of course I have no idea if there are any side effects to removing this line but as a quick fix people can just use my fork.

**Comment by domenukk** [comment]
Fixed with #14
